recipe/meta.yaml

```diff
@@ -8,6 +8,13 @@ source:
   fn: otp_src_{{ version }}.tar.gz
   url: http://erlang.org/download/otp_src_{{ version }}.tar.gz
   sha256: "fdab8129a1cb935db09f1832e3a7d511a4aeb2b9bb3602ca6a7ccb9730d5c9c3"
+  patches:
+    # Adds a patch to pickup `perl` from the path instead of
+    # forcing it to come from `/usr/bin`. See PR for details.
+    #
+    # https://github.com/erlang/otp/pull/1097
+    #
+    - PR_1097.diff
```

Comment:
This patch can be found in this PR ( https://github.com/erlang/otp/pull/1097 ). Turns out there was one place that `perl` was hardcoded and running `autoconf` didn't fix it.